Userland/Libraries/LibJS/Bytecode/Pass/MergeBlocks.cpp

```diff
@@ -48,7 +48,16 @@ void MergeBlocks::perform(PassPipelineExecutable& executable)
                 continue;
         }
 
+        {
+            InstructionStreamIterator it { entry.key->instruction_stream() };
+            while (!(*it).is_terminator())
+                ++it;
+            if ((*it).type() != Instruction::Type::Jump)
+                continue;
+        }
+
         // The two blocks are safe to merge.
+        dbgln("Setting {} to merge", entry.key->name());
```

Comment:
This and the other two dbglns should probably not be committed. Or behind a debug flag.